projects/exui-common-lib/src/lib/components/search-location/search-location.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { Observable, of } from 'rxjs';
+import { map, mergeMap } from 'rxjs/operators';
+import { LocationModel } from '../../models/location.model';
+import { LocationService } from '../../services/locations/location.service';
+
+@Component({
+  selector: 'exui-search-location',
+  templateUrl: './search-location.component.html',
+  styleUrls: ['./search-location.component.scss']
+})
+
+export class SearchLocationComponent implements OnInit {
+  @Input() public serviceId: string = '';
+  @Input() public locationType: string = '';
+  @Input() public disabled?: boolean = null;
+  @Input() public selectedLocations$: Observable<LocationModel[]>;
+  @Input() public submitted?: boolean = true;
+  @Input() public findLocationForm: FormGroup;
+
+  public locations$: Observable<LocationModel[]>;
+  public selectedLocation: LocationModel;
+  public showAutocomplete = false;
+
+  private readonly minSearchCharacters = 2;
```

Comment:
could this change to 3, and the condition change to >= minSearchCharacters, to match what requirement describes.